src/tests/run.proj

```diff
@@ -384,6 +397,75 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
     <MSBuild Projects="$(MSBuildProjectFile)" Targets="CreateXunitWrapper;BuildXunitWrapper" Properties="_CMDDIR=%(TestDirectories.Identity)" />
   </Target>
 
+  <UsingTask TaskName="AndroidAppBuilderTask" AssemblyFile="$(AndroidAppBuilderTasksAssemblyPath)"/>
```

Comment:
this needs to use a `Condition=" '$(RunWithAndroid)'=='true' "` so we only use it on android

Reply:
Fixed